user_guides/jaspic/src/main/jbake/content/intro.adoc

```diff
@@ -186,6 +186,8 @@ The {TechnologyShortName} TCK tests run on the following platforms:
 
 * Microsoft Windows 10
```

Comment:
Please remove Microsoft Windows 10 + Oracle Linux 7.1, as we didn't test with them.

Reply:
Good point, and I was already wondering about that. This one too btw is mentioned literally everywhere throughout all other specs.